mongodb/templates/mongod.conf.j2

```diff
@@ -0,0 +1,18 @@
+{% from "mongodb/map.jinja" import mongodb with context %}
+
+{# Bind to all ips(default) if in clustered mode,
+   otherwise only to the specified local ip.
+#}
+
+port = {{ mongodb.port }}
+
+# Enable journaling, http://www.mongodb.org/display/DOCS/Journaling
+{% if mongodb.enable_journal %}
+journal = true
+{% else %}
+journal = false
+nojournal = true
+{% endif %}
```

Comment:
nojournal needs to be false in order to disable journaling on 64 bit arch: https://docs.mongodb.com/v2.4/reference/configuration-options/#journal

Notice it doesn't say that we should have `nojournal` explicitly set to `false` in those docs if `journal` is set to `true`. For this reason, I kept it the same as the template logic edX used: https://github.com/edx/configuration/blob/9dad212d1048427ca86375aef7fdcfec1208d8b8/playbooks/roles/mongo/templates/mongodb.conf.j2#L21-L27